book/src/pong-tutorial/pong-tutorial-01.md

```diff
@@ -208,159 +197,54 @@ Then we call `.run()` on `game` which starts the game loop. The game will
-
-> **Note:** This setup code is directly using `rendy` crate to define the rendering.
-> You can read about its concepts in the [rendy graph docs][graph].
-
-The important thing to note is that this renders a black background.
-It is also ready to draw 2D sprites for us, which we will use in the next chapter.
+Here we are adding a `RenderingBundle`. Bundles are essentially sets of systems
+preconfigured to work together, so you don't have to write them all down one by one.
 
-If you want to use a different background color, you can tweak the RGBA
-values inside `ClearValue::Color`. Values range from `0.0` to `1.0`,
-so to get that cool green color you can try `[0.00196, 0.23726, 0.21765, 1.0]`.
-
-Now let's pack everything up and run it back in the `main()` function. We have to
-expand the existing `GameDataBuilder` with `RenderingSystem` that uses our graph:
+> **Note:** We will cover systems and bundles in more details later, for now, think of the
+> bundle as a group of functionality that together provides a certain feature to the engine.
+> You will surely be writing your own bundles for your own game's features soon.
 
-```rust,ignore
-let game_data = GameDataBuilder::default()
-    // The WindowBundle provides all the scaffolding for opening a window
-    .with_bundle(WindowBundle::from_config_path(display_config_path))?
-    // A Processor system is added to handle loading spritesheets.
-    .with(
-        Processor::<SpriteSheet>::new(),
-        "sprite_sheet_processor",
-        &[],
-    )
-    // The renderer must be executed on the same thread consecutively, so we initialize it as thread_local
-    // which will always execute on the main thread.
-    .with_thread_local(RenderingSystem::<DefaultBackend, _>::new(
-        ExampleGraph::default(),
-    ));
-
-let assets_dir = app_root.join("assets/");
+The `RenderingBundle` is actually a bit special, as it doesn't really do anything on its own,
+but relies on it's own plugin system to define what should be rendered and how. We have to use
+`with_plugin` method to tell it that we want `RenderToWindow` and `RenderFlat2D` plugins.
+Those plugins will equip our renderer with ability to open a window and draw 2D sprites on it. 
 
-let mut game = Application::new(assets_dir, Pong, game_data)?;
-game.run();
-```
+In this configuration, our window will have a black background.
+If you want to use a different color, you can tweak the RGBA
+values inside `with_clear`. Values range from `0.0` to `1.0`, to get that cool green color
+you can try `[0.00196, 0.23726, 0.21765, 1.0]`.
 
-Here we're creating a new `RenderingSystem`, adding the `ExampleGraph` we
-created. Additionally we are adding a `Processor::<SpriteSheet>` system,
-which will make sure that all `SpriteSheet` assets are being properly loaded.
-We will learn more about those in the next chapter.
+> **Note:** This setup code is using amethyst's `RenderPlugin` trait based system that
```

Comment:
Shouldn't amethyst generally be capitalized in the docs?

Reply:
Good catch! It is everywhere else.